views/forms/RegisterForm/index.jsx

```diff
@@ -0,0 +1,68 @@
+import Form from '../Form'
+import TextInput from '../TextInput'
+import { Button } from '@material-ui/core'
+
+const RegistrationForm = ({ classes, onSubmit, control, errors, navigate }) => {
+  return (
+    <Form onSubmit={onSubmit}>
+      <TextInput
+        control={control}
+        name="username"
+        label="Username"
+        required={true}
+        margin="normal"
+      />
+      <TextInput
+        control={control}
+        name="password"
+        label="Password"
+        required={true}
+        margin="normal"
```

Comment:
Can we use `<input type="password">` for the password fields?

Reply:
[c8a966a](https://github.com/AMP-SCZ/dpdash/pull/394/commits/c8a966a72451379d6eb5324fa0a3279b56ff48f5)